src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        return Optional.of(driverDao.get(id).get());
```

Comment:
you should return value from service method, not Optional. but, even in your case, `driverDao.get(id)` already returns Optional, why you extract it and wrap again?

Reply:
fixed